docs/sphinx-docs/build_sphinx.py

```diff
@@ -183,10 +193,7 @@ def apidoc():
         "-H", "SasView", # Package header
         SASVIEW_BUILD,
         # omit the following documents from the API documentation
-        joinpath(SASVIEW_BUILD, "sas", "qtgui", "GUITests.py"),
```

Comment:
removal of this exclusion causes generation of unneeded documentation for UnitTesting, for all components.

Reply:
whoops.

Reply:
that line on its own might not be the issue as that particular file doesn't exist any more. The extra `__init__.py` files might be causing it to recurse more though. I suspect a longish list of UnitTesting directories need to be added there instead.

Reply:
OK, I'll look into getting this right.

Reply:
I've added the paths, but I wonder if it might be better to get them all in the same test directory, in /test, not /src? @rozyczko - maybe you have some thoughts on this.

Reply:
Indeed. `GUITests.py` has been deleted. I didn't see the extra `__init__.py` but their presence might explain the inclusion of the test dirs.

Reply:
Originally, I put tests alongside modules, since the conversion from `wx` to `qt` was gradual and such a layout made more sense. We would convert one module, add tests, run them and go to the next module.
It might be beneficial to refactor the tests out to the top level `tests/` directory, which already contains unit tests for sascalc.

Reply:
This obviously being a separate, pretty large project, so feel free to just fix the includes here, obviously.